packages/app/src/cli/services/context/prompts.ts

```diff
@@ -41,13 +41,15 @@ export async function extensionMigrationPrompt(
     .map((name) => `"${name}"`)
     .join(', ')
 
+  const migrationEndType = toMigrate.map(({local}) => `"${local.type}"`).join(', ')
```

Comment:
Followed the same structure as the other variables in the prompt, but oddly enough this always ends up being one value in the toMigrate, even if there are two extensions to be migrated. Then if you deploy again, it will go to the second extension that needs to be migrated.